.travis.yml

```
@@ -114,10 +114,16 @@ jobs:
     - name: "cargo release linux x86_64"
       os: linux
       script:
+      # CARGO_GN_PATH and CARGO_NINJA_PATH allivate the need to build these
+      # tools from scratch. See docs at https://crates.io/crates/cargo_gn
+      - export CARGO_GN_PATH=$TRAVIS_BUILD_DIR/buildtools/linux64/gn
+      - export CARGO_NINJA_PATH=$TRAVIS_BUILD_DIR/third_party/depot_tools/ninja
       - ./tools/lint.py
       - ./tools/test_format.py
       - cargo build -vv --release --locked
       - cargo clippy --all-targets --release --locked -- -D clippy::all
+      # TODO(ry) Warning! The following does not run libdeno_test
+      - ./tools/test.py
```

```suggestion
      - DENO_BUILD_MODE=release ./tools/test.py
```
(cargo is building only the release target, so we can't test the debug one here)

The windows/appveyor failure could be related to https://github.com/mozilla/sccache/issues/264.

Also it might be worth adding a cargo mac job to the ci matrix if this is going to become the default build frontend.